src/features/customerList/__tests__/CustomerListContainer.test.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+
+import { render, screen, userEvent } from '../../../common/testUtils/testingLibrary';
+import CustomerListContainer from '../CustomerListContainer';
+
+type SetupConfig = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  mocks: any[];
+};
+
+function setup(config?: SetupConfig) {
+  const mocks = config?.mocks ?? [];
+
+  render(<CustomerListContainer />, {
+    mocks,
+  });
+
+  return {
+    toggleFilters: () => {
+      userEvent.click(screen.getByRole('button', { name: 'Rajaa' }));
+    },
```

Comment:
These helper selector could be global in this module. Then there is no need to destructure return value from `setup()`. Just like we use the global screen object in all instead of destructuring return value from render for selectors.

But this is not very important, not need to cahnge this necessarily :)